server/src/pages/delete-account/server.js

```diff
@@ -0,0 +1,39 @@
+const express = require("express");
+const reactrender = require("../../reactrender");
+const { Shot } = require("../../servershot");
+const { addDeviceActivity } = require("../../users");
+
+let app = express();
+
+exports.app = app;
+
+app.get("/", function (req, res) {
```

Comment:
Ok, I see now that this / is relative to delete-account, and /delete below is actually /delete-account/delete. I guess only the error message in the html needs to change to "You must have the addon installed to delete your account"